app/index.html

```diff
@@ -2,17 +2,22 @@
 <html lang="en">
 <head>
 <meta charset="utf-8">
-<meta http-equiv="Content-Security-Policy" content="default-src 'self' ; style-src 'self' 'unsafe-inline' ; connect-src * ; object-src 'none' ; referrer no-referrer ;">
+<meta http-equiv="Content-Security-Policy" content="default-src 'self' ; style-src 'self' 'unsafe-inline' ; connect-src * ; object-src 'none' ; img-src * ; referrer no-referrer ;">
 <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=no">
 <meta name="theme-color" content="#2F99CD">
 <meta name="format-detection" content="telephone=no">
 <meta name="msapplication-tap-highlight" content="no">
 <meta name="application-name" content="Project Link">
 <link rel="stylesheet" href="css/FiraSans/font.css">
-<link rel="stylesheet" href="css/app.css">
-<link rel="stylesheet" href="css/user-login.css">
-<link rel="stylesheet" href="css/services.css">
+<link rel="stylesheet" href="css/components/footer-menu.css">
+<link rel="stylesheet" href="css/components/modal.css">
+<link rel="stylesheet" href="css/components/service-list.css">
+<link rel="stylesheet" href="css/components/tag-list.css">
+<link rel="stylesheet" href="css/views/services/default-service.css">
+<link rel="stylesheet" href="css/views/services/ip-camera-service.css">
```

Comment:
Okay!

Just a bit of history to tell why I named it so, just as a fun story :) Some people use editors that don't have project explorer and don't indicate full path for the opened file. So they can open files with the same name, but from different directories - like ```services/ip-camera.js``` and ```controllers/ip-camera.js``` :)